### PR TITLE
Enable travis

### PR DESCRIPTION
**Pull request by Nicofuma**

### CODE REVIEW

**Comment by romqin** [comment]
:+1:

**Comment by ogizanagi** [comment]
You can configure cache with cs fixer and a config file to have the rules available for anyone using the repository.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  - 5.6
+  - 7.0
+
+install:
+  - composer self-update
+  - DATABASE_DRIVER='pdo_sqlite' DATABASE_PATH='%kernel.root_dir%/app.sqlite' composer install --prefer-source --no-interaction
+  - curl http://get.sensiolabs.org/php-cs-fixer.phar -o php-cs-fixer
+
+script:
+  - php php-cs-fixer fix src/ --dry-run -vv
```
Comment (ogizanagi) [inline .travis.yml]: `php php-cs-fixer.phar fix --no-interaction --dry-run --diff -vvv` ?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - 5.6
+  - 7.0
+
+install:
+  - composer self-update
+  - DATABASE_DRIVER='pdo_sqlite' DATABASE_PATH='%kernel.root_dir%/app.sqlite' composer install --prefer-source --no-interaction
+  - curl http://get.sensiolabs.org/php-cs-fixer.phar -o php-cs-fixer
+
+script:
+  - php php-cs-fixer fix src/ --dry-run -v
```
Comment (ogizanagi) [inline .travis.yml]: `php php-cs-fixer fix --no-interaction --dry-run --diff -vvv` ? (so you could see the diff in Travis)

**Comment by Nicofuma** [comment]
moved to php-cs-fixer

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - .php_cs.cache
```
Comment (ogizanagi) [inline .travis.yml]: I think it currently fails because of this. Not sure you can cache a file directly.

**Comment by Nicofuma** [comment]
now it should be good

**Comment by ogizanagi** [comment]
\o/ :+1:

**Comment by ogizanagi** [comment]
ping @salahm : merge ?

**Pull request merged**